js/data/locale/localematch.json

```diff
@@ -5528,7 +5528,221 @@
         "tlh": "tlh-Piqd",
         "tlh-Piqd": "tlh-Piqd",
         "tlh-PIQD": "tlh-Piqd",
-        "Piqd-PIQD": "tlh-Piqd"
+        "Piqd-PIQD": "tlh-Piqd",
+        "ar-DJ": "ar-Arab-DJ",
+        "und-ar-DJ": "ar-Arab-DJ",
+        "ar-AR-DJ": "ar-Arab-DJ",
```

Comment:
Why are there bcp-47 codes that have two regions or two languages or two scripts listed here?

Reply:
hm.. It's weird. I only added locales to likelyLocalesAdditionsl.json form like (##-##) and regenerate localematch.json by running genlikely.js. That's not I expected form.

Reply:
It looks like  genlikelyloc.js` tool code, doing more work.